tests/e2e/smoke-daemonset/00-assert.yaml

```diff
@@ -9,6 +9,10 @@ spec:
       maxSurge: 0
       maxUnavailable: 1
 status:
-  numberAvailable: 1
   numberMisscheduled: 0
-  numberReady: 1
+
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+- script: ./tests/e2e/smoke-daemonset/check-daemonset.sh
```

Comment:
Why do we need a script and cannot statically set the `desiredNumberScheduled` and `numberReady`